cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -221,10 +225,12 @@ public KafkaReconciler(
         this.pfa = pfa;
         this.imagePullPolicy = config.getImagePullPolicy();
         this.imagePullSecrets = config.getImagePullSecrets();
+        this.skipBrokerScaleDownCheck = Annotations.booleanAnnotation(kafkaCr, ANNO_STRIMZI_IO_SKIP_BROKER_SCALEDOWN_CHECK, true);
```

Comment:
are we skipping the check by default (I see the `true` default value as parameter)? Was it in the proposal? Should it be clear in the CHANGELOG as well?

Reply:
I don't think that was the plan.

Reply:
https://github.com/strimzi/proposals/pull/54/files#diff-aba8b31e792596746ba3324ed47d3810308bd8aa9a61ecfa4579b668957e2e77R64 --> I had this in the proposal

Reply:
Should I keep the check enabled by default?

Reply:
The link to the proposal does not suggest anything about it being disabled by default. The way it is written suggests it is enabled by default and you can use the annotation to disable it.

Reply:
Okay I can keep it enabled then and in the docs PR which I will create wrt to this, I will mention about how we can disable/enable this

Reply:
It seems the proposal didn't mention the default.
I think this check should be enabled by default (so here the skip annotation default should be `false`).

Reply:
It does not talk about it specifically. But the example shows how to disable the feature and not how to enable the feature. So I think it implies it to be enabled by default.

Reply:
Yeah but it's anyway ambiguous and left to interpretation. It's always better  set the expected defaults in proposal.
Anyway we are all on the same page :-)

Reply:
Done. Thanks for catching this @ppatierno